src/hint/HintBubble.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import cc from 'classcat'
+import styled from 'styled-components'
+
+import { color, radius, space, fontWeight, shadow } from '_utils/branding'
+import { A11yProps, pickA11yProps } from '_utils/interfaces'
+import Button, { ButtonStatus } from 'button'
+import CrossIcon from 'icon/crossIcon'
+
+export enum HintBubblePosition {
+  ABOVE = 'above',
+  BELOW = 'below',
+}
+
+export interface HintBubbleProps extends A11yProps {
+  title: string
+  className?: Classcat.Class
+  onClose: () => void
+  closeButtonTitle?: string
+  description?: string
+  position?: HintBubblePosition
+}
+
+export const HintBubble = (props: HintBubbleProps) => {
+  const a11yProps = pickA11yProps<HintBubbleProps>(props)
+  const { title, description, closeButtonTitle, position, className, onClose } = props
+  return (
+    <aside className={cc([className, `bubble-arrow--${position}`])} role="tooltip" {...a11yProps}>
```

Comment:
Not sure about the role="tooltip" looks like the specs is not even finished, so I'm wondering how screenreader would read this.

Reply:
Not sure about the tooltip role as well, I'll take other suggestions on how to treat this component :)

Reply:
The role="tooltip" doesn't correspond to our component behavior. You can remove it.
It's more coherent to add `aria-live="polite"` to precise there is an important dynamic content (can be removed).

Reply:
Ref for aria-live and aria-controls: https://developer.mozilla.org/fr/docs/Accessibilit%C3%A9/ARIA/Zones_live_ARIA

Reply:
And perfect for the `<aside>` tag, the screen reader explicit that it's a complementary content :-)